pkg/render/compliance_test.go

```diff
@@ -174,6 +182,14 @@ var _ = Describe("compliance rendering tests", func() {
 			Expect(dpComplianceServer.Spec.Template.Spec.Volumes[1].Secret.SecretName).To(Equal(render.ManagerInternalTLSSecretName))
 			Expect(dpComplianceServer.Spec.Template.Spec.Volumes[2].Name).To(Equal("elastic-ca-cert-volume"))
 			Expect(dpComplianceServer.Spec.Template.Spec.Volumes[2].Secret.SecretName).To(Equal(render.ElasticsearchPublicCertSecret))
+
+			clusterRole := resources[28].(*rbacv1.ClusterRole)
```

Comment:
We should use GetResource from `pkg/render/render_utils_test.go` instead of indexing in directly. It'll make the test more resilient to changing the resources.

Reply:
Thanks for making me aware of this.